Gemfile

```diff
@@ -6,6 +6,9 @@ ruby "2.7.2"
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem "rails", "6.1.3"
 
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem "webpacker"
```

Comment:
might be worth pinning this, the latest version 6 in beta introduces some breaking changes from the current stable release